cli/lib/kontena/cli/certificate/register_command.rb

```diff
@@ -12,8 +12,14 @@ def execute
       token = require_token
 
       data = {email: email}
-      response = client(token).post("certificates/#{current_grid}/register", data)
-      puts 'Email registered to LetsEncrypt'
+      puts "By registering, you agree on Let's Encrypt Terms of Service: https://letsencrypt.org/documents/LE-SA-v1.1.1-August-1-2016.pdf"
+      agree_tos = prompt.yes?("Continue?")
```

Comment:
Should there also be an `--agree-tos` flag to use for scripting?

Reply:
yes, I'll add that